.env.template

```diff
@@ -3,4 +3,6 @@ FLASK_APP=todo_app/app
 FLASK_ENV=development
 
 # Change the following values for local development.
-SECRET_KEY=secret-key
+test_key=""
+
```

Comment:
Can you add placeholders for the other variables in here too, like in .env.test?